spec/models/metric_spec.rb

```diff
@@ -427,7 +427,7 @@
     context "with a small environment and time_profile" do
       before(:each) do
         @vm1 = FactoryGirl.create(:vm_vmware)
-        @vm2 = FactoryGirl.create(:vm_vmware, :hardware => FactoryGirl.create(:hardware, :memory_mb => 4096, :cpu_sockets => 2))
+        @vm2 = FactoryGirl.create(:vm_vmware, :hardware => FactoryGirl.create(:hardware, :memory_mb => 4096, :cpu_sockets => 2, :cpu_total_cores => 2))
```

Comment:
:+1: thanks @bdunne, this is causing another test to fail.  I'll look into it, might be indicative of another problem.